scripts/common/proxy.sh

```diff
@@ -1,17 +1,21 @@
 
 function proxy_bootstrap() {
     if [ -n "$HTTP_PROXY" ]; then
+        log "HTTP proxy has the value: $HTTP_PROXY"
```

Comment:
there is a better message 3 lines below (line 7 in this case) each of these log messages.

Reply:
Yes, you are 100% right